documentation/Configuration.md

```diff
@@ -330,7 +330,7 @@ The following example shows a settings file which requires tuple element names t
   "settings": {
     "namingRules": {
       "includeInferredTupleElementNames": true,
-      "tupleElementNameCasing" : "PascalCase"
+      "tupleElementNameCasing" : "camelCase"
```

Comment:
:bulb: make sure the preceding text is consistent with this change (I can't see the full text on this small screen)

Reply:
Fixed it